2006-05-28  Aurelien Jarno <aurelien@aurel32.net>

	* linuxthreads/sysdeps/unix/sysv/linux/sparc/Makefile: pull in 
	  __syscall_error routine when building with linuxthreads.


--- /dev/null	2006-05-24 08:39:47.042929312 +0200
+++ linuxthreads/sysdeps/unix/sysv/linux/sparc/Makefile	2003-01-12 09:12:11.000000000 +0100
@@ -0,0 +1,2 @@
+# pull in __syscall_error routine
+libpthread-routines += sysdep
